Apps/AdminWebClient/src/Server/Controllers/CsvExportController.cs

```diff
@@ -97,23 +97,6 @@ public IActionResult GetNotes(DateTime? startDate = null, DateTime? endDate = nu
             return SendContentResponse("Notes", this.dataExportService.GetNotes(startDate, endDate));
         }
 
-        /// <summary>
-        /// Retrieves a list of Ratings inclusively between UTC dates if provided.
-        /// </summary>
-        /// <returns>A CSV of Ratings.</returns>
-        /// <param name="startDate">The optional start date for the data.</param>
-        /// <param name="endDate">The optional end date for the data.</param>
-        /// <response code="200">Returns the list of beta requests.</response>
-        /// <response code="401">the client must authenticate itself to get the requested response.</response>
-        /// <response code="403">The client does not have access rights to the content; that is, it is unauthorized, so the server is refusing to give the requested resource. Unlike 401, the client's identity is known to the server.</response>
-        [HttpGet]
-        [Route("GetRatings")]
-        [Produces("text/csv")]
-        public IActionResult GetRatings(DateTime? startDate = null, DateTime? endDate = null)
-        {
-            return SendContentResponse("Ratings", this.dataExportService.GetRatings(startDate, endDate));
```

Comment:
why was this removed?

Reply:
seems like the merge issue. Btw, I close this PR to to safe then create a new one